### PR TITLE
Gordon's Profile

### PR DESCRIPTION
**Pull request by gordonmotsinger**

### CODE REVIEW

**Comment by nw0428** [comment]
Looking really good!
Please remove the nmml_about_me folder.

Keep up the awesome work!

File: index.html [hunk 1]
```diff
@@ -7,7 +7,14 @@
 <body>
   <div class="center reamde">
     <h1> Spring 2015!</h1>
-    <a href="david.html">David Zabner's page</a>
+    <table>
+      <tr>
+         <a href="david.html">David Zabner's page</a>
+      </tr>
+      <tr>
```
Comment (nw0428) [inline index.html]: These two links are showing up on the same line in my browser =(

**Comment by MichelleNightward** [comment]
LGTM!

**Comment by MicahMartin** [comment]
lookgoodtome

**Comment by nleoutsa** [comment]
i can't believe you changed the blue background. SSSAAAAAAWWWWEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEET!

**Comment by nickemmons** [comment]
leonardo-dicaprio-toast-fireworks-gif

**Pull request merged**